packages/dev/core/src/node.ts

```diff
@@ -125,6 +125,14 @@ export class Node implements IBehaviorAware<Node> {
      */
     public inspectableCustomProperties: IInspectable[];
 
+    /**
+     * Gets or sets the accessibility tag to describe the mesh for accessibility purpose.
+     */
+    public accessibilityTag?: IAccessibilityTag = {

```

Comment:
A node should also notify when this element has changed so that the implementation knows it needs to update the HTML.
TBH I am not sure if the node itself is the right place or the actual implementation should cache this and check for changes, but my gut feeling is telling me it should happen in the node, similar to the way the node is caching other values and checks them. This way the HTML implementation could also know if there is an already-existing node that has its accessibility tag set.